Formula/tmuxinator.rb

```diff
@@ -0,0 +1,68 @@
+class Tmuxinator < Formula
+  desc "Manage complex tmux sessions easily"
+  homepage "https://github.com/tmuxinator/tmuxinator"
+  url "https://github.com/tmuxinator/tmuxinator/archive/v1.1.1.tar.gz"
+  sha256 "cc293578bca43ba5cf0d60c1355c6aa1da9d923a0acc274a47ceab03812a6ef4"
+  head "https://github.com/tmuxinator/tmuxinator.git"
+
+  depends_on "ruby" => "2.4.6"
```

Comment:
This is not valid syntax. And as the bot pointed out, isn't this version available for all macOS targets?

Reply:
I removed the version requirement.
If i remove the hole depends_on "ruby" line, i get this error:

```
ERROR:  Error installing tmuxinator-1.1.1.gem:
tmuxinator requires Ruby version >= 2.4.6.
```

Reply:
@SMillerDev can you explain me, what is wrong with this syntax?
I used this documentation for the depends_on part: https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md#specifying-other-formulae-as-dependencies

Reply:
You're mixing `:requirement => "version"` with `"requirement" => :stage` which are very different things.

Reply:
Thank you for explaining this. Now i understand the difference :-)

Is my change good, which makes the depends_on line valid?
I do not know, why i get the ruby version error if i skip this line.